roscpp_azure_iothub/CMakeLists.txt

```diff
@@ -123,23 +130,33 @@ endif()
 #############
 
 install(TARGETS ${PROJECT_NAME}_node
-  ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-  RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+  ARCHIVE DESTINATION lib
```

Comment:
Are there colcon equivalent for lib and bin?

Reply:
I noticed that the dynamic reconfigure cmakefile replaces the install functions with "ament_package", is there an equivalent?

Reply:
The dynamic reconfigure cmakefile still has the install function, but it's placed above `ament_package`